src/server/route/path.ts

```diff
@@ -35,8 +35,20 @@ function path(url: string): IPathRoute {
         '/tag': {
             methods: ['POST', 'GET', 'PUT', 'DELETE'],
         },
-        '/tag/proj/:proj': {
-            methods: ['GET'],
```

Comment:
The API should be:
- Get tags for project: GET /tag/:proj
— this should return both tags and the tag-node relations
- Add tag to node: POST /tag/:proj/:node
- Delete tag from node: DELETE /tag/:proj/:node/:tag
Don’t include separate endpoints for creating a tag and adding one, instead do it based on the request — no tagId -> create new label, yes tagId -> use that.

Reply:
Also, add api doc annotations.